modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -72,7 +72,7 @@ spec:
   online: true
   bootMACAddress: <NIC1-mac-address> <4>
   bmc:
-    address: <protocol>://<bmc-ip> <5>
+  address: <protocol>://<bmc-ip> <5>
```

Comment:
This indentation change is definitely a mistake.

Reply:
Updated as suggested.